src/flatc.cpp

```diff
@@ -956,9 +872,7 @@ std::unique_ptr<Parser> FlatCompiler::GenerateCode(const FlatCOptions &options,
 }
 
 int FlatCompiler::Compile(const FlatCOptions &options) {
-  if (params_.generators == nullptr || params_.num_generators == 0) {
-    return 0;
-  }
+  if (options.generators.empty()) { return 0; }
```

Comment:
Maybe add an error for no generators registered?

Reply:
Done

Reply:
Move this below binary annotation so that `--annotate` can be used.